pyanaconda/payload/__init__.py

```diff
@@ -1420,6 +1420,19 @@ def _setup_cdrom_device(self, storage, method, isodev, device):
                     method.server = host
                     method.dir = path
                 else:
+                    if flags.automatedInstall:
+                        # During a kickstart installation if we auto-detected
+                        # a local media, we should also disable the CDN in case
+                        # it was set previously and but not enough authetication
```

Comment:
redundant "and", authetication -> authentication ?

Reply:
Indeed! Fixed. :)